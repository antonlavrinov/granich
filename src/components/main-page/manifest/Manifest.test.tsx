import React from "react"
import { shallow } from "enzyme"
import { ManifestRule, PureManifest as Manifest } from "."

describe("<Manifest> component", () => {
  it("should render correctly", () => {
    expect(shallow(<Manifest image="imageUrl" />)).toMatchSnapshot()
  })
})

describe("<ManifestRule> component", () => {
  it("should render correctly", () => {
    expect(
      shallow(<ManifestRule title="title" text="text" number={3} />)
    ).toMatchSnapshot()
  })
})
