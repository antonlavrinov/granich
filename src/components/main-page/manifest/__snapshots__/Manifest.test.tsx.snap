// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Manifest> component should render correctly 1`] = `
<styled.section
  id="manifest"
>
  <styled.div>
    <styled.div>
      <styled.h2>
        Делаем из любви, а не страха
      </styled.h2>
      <styled.div>
        <styled.div>
          <Styled(Component)
            fluid="imageUrl"
            style={
              Object {
                "backgroundSize": "cover",
              }
            }
          />
          <styled.div>
            <styled.div>
              Вадим Гранич
            </styled.div>
            <styled.div>
              Создатель школы, 
              <br />
              автор курсов
            </styled.div>
          </styled.div>
        </styled.div>
        <styled.div>
          <styled.p>
            Сюда приходят, чтобы с любовью развивать в себе твердые навыки, а не из страха срочно понахвататься каких-то знаний. Поэтому, создавая курсы, я придерживаюсь трех принципов:
          </styled.p>
          <styled.div>
            <ManifestRule
              number={1}
              text="Вся информация на курсах очищена от лишнего, грамотно подана и структурирована"
              title="Выточено —"
            />
            <ManifestRule
              number={2}
              text="Курсы занимают много времени, акцент на практику (смотрите фильм Одержимость)"
              title="Интенсивно —"
            />
            <ManifestRule
              number={3}
              text="Не получится просто взять курс и забыть о нем. Дедлайны будут постоянно вас толкать вперед"
              title="С дедлайнами —"
            />
          </styled.div>
        </styled.div>
      </styled.div>
    </styled.div>
  </styled.div>
</styled.section>
`;

exports[`<ManifestRule> component should render correctly 1`] = `
<styled.div>
  <styled.span>
    title
  </styled.span>
  text
  <styled.div>
    3
  </styled.div>
</styled.div>
`;
